beacon-chain/core/helpers/signing_root.go

```diff
@@ -54,7 +53,7 @@ func ComputeSigningRoot(object interface{}, domain []byte) ([32]byte, error) {
 		if v, ok := object.(fssz.HashRoot); ok {
 			return v.HashTreeRoot()
 		}
-		return ssz.HashTreeRoot(object)
+		return [32]byte{}, errors.New("object does not ")
```

Comment:
Let's change the signature of this method to use `fssz.HashRoot` interface for the first argument.